lib/galaxy/tools/__init__.py

```diff
@@ -180,7 +180,6 @@
     "Interval_Maf_Merged_Fasta2",
     "GeneBed_Maf_Fasta2",
     "maf_stats1",
-    "Interval2Maf1",
```

Comment:
Regardless of what you're doing in the end you need to move this to `GALAXY_LIB_TOOLS_VERSIONED`.